src/views/admin/PersonDetailsView.vue

```diff
@@ -963,113 +969,107 @@
           thickness="6"
         ></v-divider>
         <!-- Two Factor Authentication -->
-        <v-container v-if="twoFactorAuthentificationStore.hasToken != undefined">
-          <v-row class="ml-md-16">
-            <v-col>
-              <h3 class="subtitle-1">{{ $t('admin.person.twoFactorAuthentication.header') }}</h3>
-              <v-row
-                class="mt-4 text-body"
-                v-if="twoFactorAuthentificationStore.hasToken"
-              >
-                <v-col
-                  class="text-right"
-                  cols="1"
-                >
-                  <v-icon
-                    icon="mdi-check-circle"
-                    color="green"
-                    v-if="twoFactorAuthentificationStore.hasToken"
-                  ></v-icon>
-                </v-col>
-                <div class="v-col">
-                  <p v-if="twoFactorAuthentificationStore.tokenKind === 'software'">
-                    {{ $t('admin.person.twoFactorAuthentication.softwareTokenIsSetUp') }}
-                  </p>
-                  <p v-if="twoFactorAuthentificationStore.tokenKind === 'hardware'">
-                    {{ $t('admin.person.twoFactorAuthentication.hardwareTokenIsSetUp') }}
-                  </p>
-                  <p v-if="twoFactorAuthentificationStore.serial">
-                    {{
-                      $t('admin.person.twoFactorAuthentication.serial') + ': ' + twoFactorAuthentificationStore.serial
-                    }}
-                  </p>
-                </div>
-              </v-row>
-              <v-row class="mt-4 text-body">
-                <v-col
-                  class="text-right"
-                  cols="1"
-                >
-                  <v-icon
-                    class="mb-2"
-                    icon="mdi-information"
-                  >
-                  </v-icon>
+        <template v-if="twoFactorAuthentificationStore.required">
```

Comment:
Haben wir 2 Endpunkte, die besagen, ob 2FA required ist oder nicht? Einmal für Übersicht und einmal Profile?

Reply:
Da wird immer geguckt, ob du einen ServiceProvider hast, der 2fa vorraussetzt. Für das Profil ist das einfach, weil man die eigenen SPs ja schon hat. Für die Übersicht habe ich einen neuen Endpunkt gebaut. Den könnte man an dieser Stelle natürlich auch benutzen, um die Logik nicht zu verstreuen, aber dafür zahlt man dann mit einem unnötigen Request ans Backend. Ich habe mich für einen Request weniger entschieden, aber könnte das noch umstellen, wenn du möchtest.

Reply:
Passt so, Ich hoffe nur die Endpunkte sind konsistent und kriege die Info von einem gleichen Single source of truth im Backend.

Reply:
Habe es jetzt doch so gemacht, dass es den extra Request gibt, aber wir dafür single source of truth haben.